libs/pmp-web/user/single-user-statistics/feature/src/lib/containers/single-user-statistics/single-user-statistics.component.ts

```diff
@@ -7,7 +7,7 @@ import {
 } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { first } from 'rxjs/operators';
-import { SingleUserStatisticsResponse } from '@pimp-my-pr/shared/domain';
+import { PrStatistics, SingleUserStatisticsResponse } from '@pimp-my-pr/shared/domain';
```

Comment:
Not used import.

Reply:
Is used in line 37:32 by `SingleUserStatisticsComponent.onNavigateItem` method